anvil_consortium_manager/adapters/default.py

```diff
@@ -11,6 +11,7 @@ class DefaultAccountAdapter(BaseAccountAdapter):
 
     list_table_class = tables.AccountStaffTable
     list_filterset_class = filters.AccountListFilter
+    ACCOUNT_LINK_VERIFY_MESSAGE = "Thank you for linking your AnVIL account."
```

Comment:
Change this to `account_link_verify_message` for case consistency adapter attributes